These symbols are different when compiling libgcc with --enable-sjlj-exceptions

--- libgcc/config/rs6000/libgcc-darwin.10.4.ver.orig
+++ libgcc/config/rs6000/libgcc-darwin.10.4.ver
@@ -19,7 +19,7 @@
 __Unwind_DeleteException
 __Unwind_FindEnclosingFunction
 __Unwind_Find_FDE
-__Unwind_ForcedUnwind
+__Unwind_SjLj_ForcedUnwind
 __Unwind_GetCFA
 __Unwind_GetDataRelBase
 __Unwind_GetGR
@@ -27,9 +27,9 @@
 __Unwind_GetLanguageSpecificData
 __Unwind_GetRegionStart
 __Unwind_GetTextRelBase
-__Unwind_RaiseException
-__Unwind_Resume
-__Unwind_Resume_or_Rethrow
+__Unwind_SjLj_RaiseException
+__Unwind_SjLj_Resume
+__Unwind_SjLj_Resume_or_Rethrow
 __Unwind_SetGR
 __Unwind_SetIP
 ___absvdi2
@@ -61,7 +61,7 @@
 ___floatdidf
 ___floatdisf
 ___floatditf
-___gcc_personality_v0
+___gcc_personality_sj0
 ___gcc_qadd
 ___gcc_qdiv
 ___gcc_qmul
--- libgcc/config/rs6000/libgcc-darwin.10.5.ver.orig
+++ libgcc/config/rs6000/libgcc-darwin.10.5.ver
@@ -19,7 +19,7 @@
 __Unwind_DeleteException
 __Unwind_FindEnclosingFunction
 __Unwind_Find_FDE
-__Unwind_ForcedUnwind
+__Unwind_SjLj_ForcedUnwind
 __Unwind_GetCFA
 __Unwind_GetDataRelBase
 __Unwind_GetGR
@@ -28,9 +28,9 @@
 __Unwind_GetLanguageSpecificData
 __Unwind_GetRegionStart
 __Unwind_GetTextRelBase
-__Unwind_RaiseException
-__Unwind_Resume
-__Unwind_Resume_or_Rethrow
+__Unwind_SjLj_RaiseException
+__Unwind_SjLj_Resume
+__Unwind_SjLj_Resume_or_Rethrow
 __Unwind_SetGR
 __Unwind_SetIP
 ___absvdi2
@@ -68,7 +68,7 @@
 ___floatundidf
 ___floatundisf
 ___floatunditf
-___gcc_personality_v0
+___gcc_personality_sj0
 ___gcc_qadd
 ___gcc_qdiv
 ___gcc_qmul
